client/src/featureform/register.py

```diff
@@ -4561,23 +4569,24 @@ def register(self):
         )
 
     def features_and_labels(self) -> Tuple[List[ColumnMapping], List[ColumnMapping]]:
```

Comment:
wtf is this method used for lol

Reply:
When I implemented the class API and was debugging some of the logic, I got annoyed with having the either/or logic of features/labels in my way, so I pushed it to another method. I obviously should name this something better than what I decided on at the time.